examples/unadf.c

```diff
@@ -305,20 +309,20 @@ void processFile(struct Volume *vol, char* name, char* path, unsigned char *extb
     }
     else {
         /* the all-in-one string : to call system(), to find the filename, the convert dir sep char ... */
```

Comment:
True, thanks. I'll force-update the pull request to just update the comments. I'll remove that comment and modify the comment about portability at the top of the file.

p.s. `extractTree()` needs quite a bit cleaning/refactoring, but that needs to be done separately.